.github/workflows/booster-schedule.yaml

```
@@ -0,0 +1,56 @@
+name: Boost backers' rewards in an active NFT campaign
+
+permissions:
+  contents: write
+
+env:
+  BOOST: '30'
+  NFT_CONTRACT: '0x1EEA464C8632A71A2E9B364725da6e09aaed2Ddc'
+  ENVIRONMENT: 'mainnet'
+  BOOST_DATA_BRANCH: 'boost_data/v1.7.0'
+
+on:
+  schedule:
+    - cron: '0 12 * * *'  # Runs every day at noon UTC
+
+jobs:
+  boost:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

Could we specify the exact commit? 
```suggestion
        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
```